images/linux/scripts/installers/codeql-bundle.sh

```diff
@@ -4,22 +4,37 @@
 ##  Desc:  Install the CodeQL CLI Bundle to the toolcache.
 ################################################################################
 
-source $HELPER_SCRIPTS/install.sh
+source "$HELPER_SCRIPTS/install.sh"
 
 # Retrieve the name of the CodeQL bundle preferred by the Action (in the format codeql-bundle-YYYYMMDD).
-codeql_bundle_name="$(curl -sSL https://raw.githubusercontent.com/github/codeql-action/v2/src/defaults.json | jq -r .bundleVersion)"
-# Convert the bundle name to a version number (0.0.0-YYYYMMDD).
-codeql_bundle_version="0.0.0-${codeql_bundle_name##*-}"
+defaults="$(curl -sSL https://raw.githubusercontent.com/github/codeql-action/v2/src/defaults.json)"
+codeql_bundle_name="$(${defaults} | jq -r .bundleVersion)"
+codeql_cli_version="$(${defaults} | jq -r .cliVersion)"
+prior_codeql_bundle_name="$(${defaults} | jq -r .priorBundleVersion)"
+prior_codeql_cli_version="$(${defaults} | jq -r .priorCliVersion)"
```

Comment:
Hmm, this looks like it'll try to execute `defaults.json`.  I think we might want to do `jq -r '.bundleVersion' <<< "${defaults}"` instead.

Reply:
Ah, that makes sense

Reply:
Done!